app/containers/wallet/dialogs/WalletTrezorConnectDialogContainer.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+
+import environment from '../../../environment';
+import type { InjectedDialogContainerProps } from '../../../types/injectedPropsType';
+
+import AboutDialog from '../../../components/wallet/trezorConnect/AboutDialog';
+import ConnectDialog from '../../../components/wallet/trezorConnect/ConnectDialog';
+import SaveDialog from '../../../components/wallet/trezorConnect/SaveDialog';
+
+import { ProgressStepOption } from '../../../stores/ada/TrezorConnectStore';
+
+type Props = InjectedDialogContainerProps;
+@inject('stores', 'actions') @observer
+export default class WalletTrezorConnectDialogContainer extends Component<Props> {
+
+  static defaultProps = { actions: null, stores: null, children: null, onClose: () => {} };
+
+  cancel = () => {
+    this.props.onClose();
+    this._getAction().cancel.trigger();
+  };
+
+  render() {
+    const store = this._getStore();
+    const action = this._getAction();
+
+    let component = null;
+
+    switch (store.progressInfo.currentStep) {
+      case ProgressStepOption.ABOUT:
+        component = (
+          <AboutDialog
+            progressInfo={store.progressInfo}
+            isActionProcessing={store.isActionProcessing}
+            error={store.error}
+            submit={action.submitAbout.trigger}
+            cancel={this.cancel}
+          />);
+        break;
+      case ProgressStepOption.CONNECT:
+        component = (
+          <ConnectDialog
+            progressInfo={store.progressInfo}
+            isActionProcessing={store.isActionProcessing}
+            error={store.error}
+            goBack={action.goBacktToAbout.trigger}
+            submit={action.submitConnect.trigger}
+            cancel={this.cancel}
+          />);
+        break;
+      case ProgressStepOption.SAVE:
+        component = (
+          <SaveDialog
+            progressInfo={store.progressInfo}
+            isActionProcessing={store.isActionProcessing}
+            error={store.error}
+            defaultWalletName={store.defaultWalletName}
+            submit={action.submitSave.trigger}
+            cancel={this.cancel}
+          />);
+        break;
+      default:
+        console.error('Error : something unexpected happened');
```

Comment:
use `Logger`

Reply:
Better error’ing. If you read this on the console, it doesn‘t give enough information. Where was this call? what were the unexpected value?

Reply:
DONE